src/deltaSync/types.ts

```diff
@@ -2,12 +2,14 @@ import { User } from '../users/types';
 import { Task } from '../tasks/types';
 import { Project } from '../projects/types';
 import { TimeTracking } from '../timetrackings/types';
+import { UserStatusOverview } from '../userStatusOverview/types';
 
 export type SyncData = {
   users: User[];
   tasks: Task[];
   projects: Project[];
   timeTrackings: TimeTracking[];
+  userStatusOverviews: UserStatusOverview[];
```

Comment:
Why is this needed?

Reply:
will recheck this one

Reply:
because status overview will be periodically sync anyway. if approach of syncing is changed in future then it should be removed from here, but for now I think it is ok

Reply:
removed as it was already removed from FE repo